### DISCUSSION TITLE
initial run fails as $MAGMA_ROOT is an empty directory

### DISCUSSION THREAD

**Discussion by mugaizzo** [magma/magma#11286] [Q&A]
hello everyone, 
   I'm trying to make the inital run following the [initial run documentation](https://docs.magmacore.org/docs/basics/quick_start_guide#initial-run) , i get below result:
   
![image](https://user-images.githubusercontent.com/15838537/151227460-13a17d2d-5e54-4069-85d6-2a21b566a8c1.png)

the magma folder is empty, did i miss anything?

**Comment by sebathomas** [comment]
It looks like your magma repo is not available inside the VM. Normally `$MAGMA_ROOT` should contain a link to the repository on your hard disk. Did you install all the prerequisites from https://docs.magmacore.org/docs/basics/prerequisites, e.g. the vagrant plugin?
